### DISCUSSION TITLE
Enhanced Logging

### DISCUSSION THREAD

**Discussion by mrtoner** [derekantrican/GAS-ICS-Sync#342] [Q&A] [answered]
I'm getting an error that's, well, keeping the script from completing:

9:42:23 AM	Info	Updating existing event 3E0C3586-FD6A-43C0-A8E1-A447D7C3FFDF
9:42:23 AM	Error	
API call to calendar.events.update failed with error: Invalid Value

Is there some enhanced logging I can enable (maybe just a line to add to the script for the moment) that would help me identify the event that's causing the issue? I suspect there's just something in an event I can correct or perhaps just delete the event.

**Comment by jonas0b1011001** [comment, marked as answer]
> 9:42:23 AM Info Updating existing event 3E0C3586-FD6A-43C0-A8E1-A447D7C3FFDF

That should actually be enough to identify the event. Open your source ics in a text editor an search for _3E0C3586-FD6A-43C0-A8E1-A447D7C3FFDF_.

**Reply by mrtoner** [reply]
Jonas, it was indeed as easy as that! Thank!
